### ISSUE TITLE
dependency conflict, unable to install

### ISSUE THREAD

**Issue by nikisalli** [ros-noetic-arch/ros-noetic-python-qt-binding#9]
this package wants `python-pyqt5-sip4` but other ros noetic packages want `python-pyqt5`, and this two conflict with each other

**Comment by acxz** [comment]
Notice that `python-pyqt5-sip4` provides for `python-pyqt5`: https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=python-pyqt5-sip4#n32

Any package that requires `python-pyqt5` will work if `python-pyqt5-sip4` is installed instead.

If you post your error message, I can better help you.

**Comment by nikisalli** [comment]
`python-pyqt5-sip4` doesn't even build correctly:
```
Traceback (most recent call last):
  File "/usr/bin/sip-build", line 33, in <module>
    sys.exit(load_entry_point('sip==6.6.2', 'console_scripts', 'sip-build')())
  File "/usr/bin/sip-build", line 25, in importlib_load_entry_point
    return next(matches).load()
StopIteration
```
I'd prefer to keep using `python-pyqt5` because it is in the community repo and it looks better mantained, isn't it possible to make this package use it?

**Comment by acxz** [comment]
> python-pyqt5-sip4 doesn't even build correctly:

See the [pinned issue](https://github.com/acxz/pkgbuilds/issues/175) at corresponding repo for that package.

> possible to make this package use it?

I'm afraid not currently, maybe in the future though, see https://github.com/ros-noetic-arch/ros-noetic-python-qt-binding/issues/7 for further discussion on that topic.

**Comment by acxz** [comment]
This issue is resolved with 2c51094, i.e the removal of python-pyqt5-sip4 dep

**Comment by acxz** [comment]
Closing this issue due to inactivity,
@nikisalli if you are encountering any specific issues with dep conflicts please feel free to comment and I'll open the issue again.
